### ISSUE TITLE
systemsettings5 showing two panels for monitor settings: "display" and "display configuration"

### ISSUE THREAD

**Issue by slartibart70** [ZaWertun/fedora-copr-kde5#134]
Hi, 

i opened this bug
https://bugs.kde.org/show_bug.cgi?id=458089
recently

Anything you might be able to do (as i am using the zawertun repos)?
Thanks in advance

**Comment by ZaWertun** [comment]
Can you paste the output of `kcmshell5 --list`?

**Comment by ZaWertun** [comment]
Maybe related: https://bugs.kde.org/show_bug.cgi?id=449563.

**Comment by ZaWertun** [comment]
I've got it!
You have installed `kdisplay` package from kwinft project (KWin fork, I moved it to kwinft copr).
After installing `kdisplay` I've got duplicate screen settings too.
I could make `kdisplay` to conflict with `kscreen`.

**Comment by ZaWertun** [comment]
> I've got it! You have installed `kdisplay` package from kwinft project (KWin fork, I moved it to kwinft copr). After installing `kdisplay` I've got duplicate screen settings too. I could make `kdisplay` to conflict with `kscreen`.

TLDR: To solve this issue remove `kdisplay` package.

**Comment by slartibart70** [comment]
Hi,
sorry, couldn't reply earlier.
And yes, you are right - kdisplay was indeed installed on this machine.
Strangely enough, i did not play with kwinft on this laptop, so i'm not sure how this was installed (i would say, never intentionally)

Anyway, thanks for your help
i removed kdisplay and all is fine again!
